pkg/asset/installconfig/azure/client.go

```diff
@@ -286,11 +286,15 @@ func (c *Client) GetVMCapabilities(ctx context.Context, instanceType, region str
 	if typeMeta == nil {
 		return nil, fmt.Errorf("not found in region %s", region)
 	}
-
+	if typeMeta.Family == nil {
+		return nil, fmt.Errorf("error getting resource family")
```

Comment:
Should we fail/prevent an install in this case? If we can’t determine the family we don’t know it will fail, so perhaps a warning is better.

Reply:
I've rearranged the code so that the error is ignored.